### PR TITLE
update elb_xxxx module adding ip_address_type option

### PR DESCRIPTION
**Pull request by abikouo**
##### SUMMARY

This is a new feature to add the new option ip_address_type for elb_xxx modules

Feature requested on issue https://github.com/ansible-collections/community.aws/issues/397
Depends on https://github.com/ansible-collections/amazon.aws/pull/310
##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

elb_application_lb
elb_application_lb_info
elb_network_lb
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by jillr** [comment]
recheck

File: tests/integration/targets/elb_application_lb_info/tasks/setup.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    register: subnets
+
+  - ec2_vpc_subnet_info:
+      filters:
+        vpc-id: '{{ vpc.vpc.id }}'
+    register: vpc_subnets
+
+  - name: create list of subnet ids
+    set_fact:
+      alb_subnets: '{{ vpc_subnets|community.general.json_query(''subnets[?tags.Public == `True`].id'') }}'
```
Comment (jillr) [inline tests/integration/targets/elb_application_lb_info/tasks/setup.yml:49]: Not a blocker but nice to have, I'd like to eventually remove or reduce the CI dependency on community.general in the AWS collections so I've started switching to selectattr in tests that currently use json_query.
Example in tests/integration/targets/ec2_vpc_route_table/tasks/main.yml: https://github.com/ansible-collections/community.aws/pull/484/files

**Review by jillr — COMMENTED** [review]
A few small comments, the new policies have been deployed to CI

File: tests/integration/targets/elb_application_lb/tasks/test_alb_ip_address_type_options.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      subnets: "{{ alb_subnets }}"
+      security_groups: "{{ sec_group.group_id }}"
+      state: present
+      listeners:
+        - Protocol: HTTP
+          Port: 80
+          DefaultActions:
+            - Type: forward
+              TargetGroupName: "{{ tg_name }}"
+      ip_address_type: "ipv6"
```
[the remark this answers is not in the document]
Comment (abikouo) [reply tests/integration/targets/elb_application_lb/tasks/test_alb_ip_address_type_options.yml:18]: @jillr totally agree

**Comment by jillr** [comment]
recheck

**Review by jillr — APPROVED** [review]
thanks @abikouo!

**Pull request merged**